pruning_planting.py

```diff
@@ -0,0 +1,95 @@
+"""
+Data processing for tree data.
```

Comment:
Would be nice if there was documentation at the top that specified the expected input and output of the data formats, but you don't have to do it now.

Reply:
Added some documentation.